planner/core/logical_plan_builder.go

```diff
@@ -189,11 +189,19 @@ func (b *PlanBuilder) buildResultSetNode(ctx context.Context, node ast.ResultSet
 			return nil, err
 		}
 
+		_, isVirtualTable := p.(*LogicalProjection)
 		for _, name := range p.OutputNames() {
 			if name.Hidden {
 				continue
 			}
-			name.OrigTblName = name.TblName
+			if !isVirtualTable {
```

Comment:
I think we just don't change the original table name here is okay?

Reply:
@winoros Comment addressed, PTAL